### ISSUE TITLE
Suggest putting continuation indicators at the start of the line

### ISSUE THREAD

**Issue by l0b0** [koalaman/shellcheck#2596]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet that shows the problem:

```sh
#!/usr/bin/env bash
grep … | \
    sed … | \
    cut … || \
    echo …
```

#### Here's what shellcheck currently says:

> No issues detected!

#### Here's what I wanted or expected to see:

> Put continuation indicators at the start of the line:

```shell
grep … \
    | sed … \
    | cut … \
    || echo …
```

Rationale: It's easier to tell apart the various command separators if they are lined up and separate from other distracting syntax like `\`. [Black does the same thing](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html) for Python.

**Comment by koalaman** [comment]
Check out `shfmt` which supports both styles (`shfmt -bn` for the latter), and auto-formats them better than ShellCheck could. For the most part, ShellCheck focuses on issues related to semantics and does not try to arbitrate style.
